kuma/users/jinja2/users/user_edit.html

```diff
@@ -24,7 +24,7 @@
 {% endblock %}
 
 {% block content %}
-  <form class="submission" method="post" action="">
+
     {% csrf_token %}
```

Comment:
The ``{% csrf_token %}`` needs to be moved down inside the submission form, or the user gets a Permission Denied on profile edits.